resources/js/components/WandSelectorComponent.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <div class="row">
+      <div class="form-group col-auto">
+        <label for="wand_length">Wand Length:</label>
+        <select class="form-control" name="wand_length" v-model="selected_length" id="wand_length" :disabled="is_magical">
+          <option disabled value="">Select Length</option>
+          <option v-for="lengthOption in wand_lengths"
+                  :value="lengthOption"
+                  :key="lengthOption"
+                  :selected="lengthOption === selected_length">{{ lengthOption }} inches</option>
+        </select>
+      </div>
+
+      <div class="form-group col">
+        <label for="wand_wood">Wand Wood:</label>
+        <select class="form-control" name="wand_wood" v-model="selected_wood" id="wand_wood" :disabled="is_magical">
+          <option disabled value="">Select Wood</option>
+          <option v-for="woodOption in wood_list"
+                  :value="woodOption.id"
+                  :key="woodOption.id"
+                  :selected="woodOption.id === selected_wood">{{ woodOption.name }}</option>
+        </select>
+      </div>
+
+      <div class="form-group col">
+        <label for="wand_core">Wand Core:</label>
+        <select class="form-control" name="wand_core" v-model="selected_core" id="wand_core" :disabled="is_magical">
+          <option disabled value="">Select Core</option>
+          <option v-for="coreOption in core_list"
+                  :value="coreOption.id"
+                  :key="coreOption.id"
+                  :selected="coreOption.id === selected_core">{{ coreOption.name }}</option>
+        </select>
+      </div>
+    </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+export default {
+  name: "WandSelectorComponent",
+  props: ['wood_list', 'core_list', 'wood', 'core', 'length'],
+  data() {
+    return {
+      selected_length: '',
+      selected_core: '',
+      selected_wood: '',
+      wand_lengths: []
+    }
+  },
+  mounted() {
+    if(this.length) this.selected_length = this.length;
+    if(this.core) this.selected_core = this.core;
+    if(this.wood) this.selected_wood = this.wood;
+  },
+  computed: {
+    ...mapGetters([
+      'blood_status'
+    ]),
+    is_magical() {
+      return (
+          this.blood_status === 'Squib' ||
+          this.blood_status === 'Muggle'
+      );
+    }
+  },
+  watch: {
+    blood_status: function(val) {
+      if(val === 'Squib' || val === 'Muggle') {
+        this.selected_core = '';
+        this.selected_wood = '';
+        this.selected_length = '';
+      }
+    }
+  },
+  created() {
+      for(let i = 5.00; i <= 20.00; i++) {
```

Comment:
bit short for a wand - the shortest want I can find is 10.5, scrolling through so much is a little annoying

```suggestion
      for(let i = 10.00; i <= 20.00; i++) {
```